build/bootstrap/bootstrap-debian.sh

```diff
@@ -32,6 +32,9 @@ sudo apt-get install -y --no-install-recommends \
 # pnpm doesn't provide a Debian repository, and supports either `curl | sh` or `npm install -g` installations.
 curl -fsSL https://get.pnpm.io/install.sh | env PNPM_VERSION=8.6.6 sh -
 
+# append new line to bashrc
+echo -e '\n' >> ~/.bashrc
```

Comment:
How about either moving this to before line 42 (`echo '. ~/.bashrc_bootstrap' >> ~/.bashrc`) where it's necessity is clear on its own or stating in the comment why the new line is necessary? (If I were doing it, I'd move it to be before line 42.)

Reply:
I see. That is true. But I'd also expect they would be testing it to make sure it works as expected, so they'd catch it. 

Another thing that would make it easier to understand the generated file would be to have a blank line between the pnpm installed code and the `. ~/.bashrc_bootstrap` line. That makes it easier to see the blocks of functionality and has a visual separation of the sourcing of .bashrc_bootstrap from the pnpm code. So putting a blank line in both places would be a good idea. What do you think of this idea?

Reply:
I like this idea! I will fix this.